plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/NewNativeAppEngineStandardProjectTest.java

```diff
@@ -49,27 +49,29 @@
 
   @Test
   public void testWithDefaults() throws Exception {
+    Assume.assumeTrue("Requires a Java 8 JRE", JavaRuntimeUtils.hasJavaSE8());
     String[] projectFiles = {"src/main/java/HelloAppEngine.java",
         "src/main/webapp/META-INF/MANIFEST.MF", "src/main/webapp/WEB-INF/appengine-web.xml",
         "src/main/webapp/WEB-INF/web.xml", "src/main/webapp/index.html"};
-    createAndCheck("appWithDefault", null /* packageName */, null /* runtime */, projectFiles);
+    createAndCheck("appWithDefault_java8", null /* packageName */, null /* runtime */,
```

Comment:
maybe better as `_default`?

Reply:
`...Default_default` sounds weird, and I think it's only more informative to say `_java8`.